documentation/frontend/guides/quickstart.md

```diff
@@ -57,7 +57,7 @@ you need to run this.
    use the local API with the frontend.
 
    ```bash
-   env API_URL="http://localhost:50280" just frontend/run dev
+   env API_URL="http://localhost:50280" ./ov just frontend/run dev
```

Comment:
`env` here is running on the host, and won't be available to the commands inside `ov`. Instead, we need to put `env` after `ov`:

```suggestion
./ov env API_URL="http://localhost:50280" just frontend/run dev
```